src/hapi/pipelines/app/pipelines.py

```diff
@@ -161,12 +171,6 @@ def _create_configurable_scrapers(
             "operational_presence", "adminone", adminlevel=self.adminone
         )
         _create_configurable_scrapers("operational_presence", "national")
-        _create_configurable_scrapers(
```

Comment:
Good question! The configurable scraper handles wide data better than long data although on reflection it may be possible to use the `list:` option to handle the issue where one admin unit maps to multiple values in a column inherent in long data. 

While there is a wide version of the IPC data, handling wide data makes the population of the database much more complex than long data which matches how HAPI wants it. 

It may make sense to make a new configurable scraper in HDX Python Scraper that is specifically for long data and outputs a long rather than wide data structure (or perhaps it would be better if it's in HAPI Pipelines itself), although for IPC there is additional logic for handling admin codes that make a custom scraper necessary.